modules/@apostrophecms/notification/index.js

```diff
@@ -156,22 +156,19 @@ module.exports = {
       // Resolves with an object with `notifications` and `dismissed`
       // properties.
       //
-      // If `options.displayingIds` is set, notifications
-      // whose `_id` properties appear in it are not returned.
+      // If `options.modifiedOnOrSince` is set, notifications
+      // greater than the timestamp are sent.
 
       async find(req, options) {
         try {
           const notifications = await self.db.find({
-            userId: req.user._id
+            userId: req.user._id,
+            ...(options.modifiedOnOrSince && { createdAt: { $gt: new Date(options.modifiedOnOrSince) } })
```

Comment:
`updatedAt`. We'll need to start establishing `updatedAt` as we start rolling out progress meter notifications, which have status info that changes over their lifetime.

Reply:
There's more in the technical design on this but I get why it's just createdAt for now, you're not into the activity log window stuff yet.